src/logic/useMapPoints.js

```diff
@@ -1,14 +1,19 @@
 import { useState, useEffect } from 'react';
 import { collection, onSnapshot } from 'firebase/firestore';
 import { db } from 'logic/fb';
+import firebase from 'firebase/compat/app';
```

Comment:
Dziwny import?

Reply:
Wymagany jest do konstruktora obiektu typu geopoint poniżej

Reply:
Na pewno compat?

Reply:
A ma być jakoś inaczej?, tak gdzieś znalazłem że powinno sie to importować ale nie wiem

Reply:
https://stackoverflow.com/questions/68946446/how-do-i-fix-a-firebase-9-0-import-error-attempted-import-error-firebase-app

Reply:
Dziwne bo to jest jakby deprecated. Nie ma nowej metody na dostanie się do GeoPoint? Znając Google to wszystko jest możliwe więc whatever, może zostać.